src/otx/algorithms/detection/adapters/mmdet/datasets/pipelines/load_pipelines.py

```diff
@@ -41,22 +41,27 @@ class LoadAnnotationFromOTXDataset:
 
     def __init__(
         self,
-        min_size: int,
+        min_size: int = -1,
         with_bbox: bool = True,
         with_label: bool = True,
         with_mask: bool = False,
         with_seg: bool = False,
         poly2mask: bool = True,
         with_text: bool = False,
-        domain: Optional[Domain] = None,
+        domain: Optional[str] = "detection",
```

Comment:
```suggestion
        domain: str = "detection",
```
domain can't be `None`, right?